LogReader/LogReader/Models/Log.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+
+namespace LogReader.Models
+{
+    public class Log
```

Comment:
Why do you have 2 identical models with different names (LogMessage and Log)?

Reply:
> Why do you have 2 identical models with different names (LogMessage and Log)?

Log was automatically generated after database scaffolding, then didn't realize that it's better to use old LogMessage, changed it